brouter-util/src/main/java/btools/util/CheapRulerSingleton.java

```diff
@@ -33,7 +33,7 @@
   }
 
   private static double[] calcKxKyFromILat(int ilat) {
-    double lat = DEG_TO_RAD*(ilat-90000000)/1000000.;
+    double lat = DEG_TO_RAD*ilat*ILATLNG_TO_LATLNG - 90;
```

Comment:
Just a little rewrite to make it a bit more readable and remove magic constants, in my opinion.